docs/DeveloperGuide.md

```diff
@@ -118,7 +118,7 @@ Here's a (partial) class diagram of the `Logic` component:
 How the `Logic` component works:
 1. When `Logic` is called upon to execute a command, it uses the `AddressBookParser` class to parse the user command.
```

Comment:
Should this be GuestBookParser, or do we not consider refactoring the classes yet? Similar cases at e.g Line 198, etc... about changing AddressBook -> GuestBook?